lib/matplotlib/tests/test_lines.py

```diff
@@ -7,7 +7,7 @@
 import six
 
 import nose
-from nose.tools import assert_true, assert_raises
+from nose.tools import assert_true, assert_raises, assert_raises_regexp
```

Comment:
I can see keeping py2.6 for just a short while longer (maybe just for the
upcoming couple of releases?). But, I don't see 2.7 going away anytime
soon, unfortunately. We probably won't be one of those straggler projects
that holds on until 2020, but with the exception of keyword-only arguments,
I haven't seen much in py3k that is relevant to this project. There are
lots of goodies in py3k, I just haven't seen anything yet that made me want
to go "ooh, I really want that for feature X in matplotlib!".

On Tue, Jun 23, 2015 at 4:11 PM, Jens Hedegaard Nielsen <
notifications@github.com> wrote:

> In lib/matplotlib/tests/test_lines.py
> <https://github.com/matplotlib/matplotlib/pull/3195#discussion_r33085846>:
>
> > @@ -7,7 +7,7 @@
> >  import six
> >
> >  import nose
> > -from nose.tools import assert_true, assert_raises
> > +from nose.tools import assert_true, assert_raises, assert_raises_regexp
>
> Not sure that it's relevant but IPython dropped support some time ago with
> version 2.0
>
> More to the point I guess we can maintain support for it as long as we can
> test it on Travis and the work arounds are limited to minor issues like
> this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/3195/files#r33085846>.
>